scipy/optimize/_minimize.py

```diff
@@ -833,6 +833,12 @@ def minimize_scalar(fun, bracket=None, bounds=None, args=(),
     perform bounded minimization [2]_ [3]_. It uses the Brent method to find a
     local minimum in the interval x1 < xopt < x2.
 
+    Note that the Brent and Golden methods do not guarantee success unless a
+    valid `bracket` triple is provided. If a three-point bracket cannot be
```

Comment:
Minor point, this now links to the `bracket` function not sure if this was intended or not

Reply:
Nope, I was just following numpydoc standards for single vs double backticks. I'll change it, thanks!

Reply:
```suggestion
    valid ``bracket`` triple is provided. If a three-point bracket cannot be
```